tests/api/routes/pipelines.test.js

```diff
@@ -110,8 +109,8 @@ describe('PipelineResults route', () => {
     expect(https.get).toHaveBeenCalledTimes(1);
   });
 
-  it('Get malformatted work results returns an error', async () => {
-    const brokenMsg = JSON.stringify();
+  it('Returns an error for malformed work', async () => {
```

Comment:
Just noticed - is this test actually any different to the one starting on line 63? :thinking:

Reply:
Hmm brokenMsg and validMsg seem to have different properties

Reply:
Yes, they are invalid in different ways (`invalidMsg` is missing the `Type` field whereas `brokenMsg` is just completely empty). I will leave both as I don't want to remove a test without fully understanding the reasons both were added.